### ISSUE TITLE
since updating discord to x64 (by default discord used to force install the x86 version) i get stuck

### ISSUE THREAD

**Issue by ElectroYModz** [andro2157/DiscordTokenProtector#87]
**Describe the bug**
It gets stuck on "Injecting payload"

**To Reproduce**
1. install discord x64 (https://discord.com/api/downloads/distributions/app/installers/latest?channel=stable&platform=win&arch=x64)
2. start Discord Token Protector
3. Enter your password

**Expected behavior**
not get stuck

**Screenshots**
![image](https://github.com/andro2157/DiscordTokenProtector/assets/44715385/9c30ed35-ada2-41c2-b523-df891556d58b)

**Setup (please complete the following information):**
- OS Version and build: Windows 10 Enterprise 22H2 19045.4291
- Discord Token Protector Version: dev-11
- Antivirus: ESET

**Logs:**
Is there a file called "logging.log" in `%appdata%\DiscordTokenProtector` ? If so can you send it here?
[logging.log](https://github.com/andro2157/DiscordTokenProtector/files/15026298/logging.log)

**Comment by Surioxy** [comment]
Yes, I can confirm this, I have exactly the same problem with the "injection payload" message which does nothing and in the end discord doesn't light up and I have to end the Discord token protector task because the software crashes.

**Comment by KiSki-Dev** [comment]
Yes, I also confirm this. I had this issue first time today. I dont know why I got so late but it stucks itself at "Injection Payload". Nothing helps, hoping on a Fix!

**Comment by ghost** [comment]
Compile ProtectionPayload in x64. Error is Discord::injectPayload tries to inject 32-bit dll into 64-bit Discord. LoadLibrary will fail

**Comment by GlitchedCod** [comment]
Same here !
